ui/tests/acceptance/policies/index-test.js

```diff
@@ -97,6 +97,7 @@ module('Acceptance | policies/acl', function (hooks) {
   });
 
   test('it can create and delete correctly', async function (assert) {
+    const flashSuccessSpy = sinon.spy(this.owner.lookup('service:flash-messages'), 'success');
```

Comment:
I've experienced a lot of flakiness using sinon to check flash messages. My latest approach has been to check the contents of `GENERAL.latestFlashContent` selector, which seems to work pretty well. If it's passing consistently locally that might not be necessary, although it's nice to have consistency in how we check these.

Reply:
It seemed to past consistently locally, but I can update so we assert these the same!